<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>key_setting_dialog</class>
 <widget class="QDialog" name="key_setting_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>108</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Public Key</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Private Key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pubKeyEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="privKeyEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>40</y>
     <width>311</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="keyEditBtn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>70</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>변경</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
